interface/src/app/node/node.tpl.html

```diff
@@ -7,13 +7,21 @@
   <div class="container">
     <div class="row">
       <div class="col-xs-6">
-        <high-weight-genes node-id="{{ctrl.id}}"></high-weight-genes>
+        <high-weight-genes node-id="{{ctrl.id}}" h-w-genes="ctrl.hWGenes"></high-weight-genes>
```

Comment:
I decided to just go with "genes", and share that in the scopes across all the directives that use them in this Node page.